nf_core/modules/pipeline_modules.py

```diff
@@ -231,7 +236,8 @@ def install(self, module=None):
         files = self.modules_repo.get_module_file_urls(module)
         log.debug("Fetching module files:\n - {}".format("\n - ".join(files.keys())))
         for filename, api_url in files.items():
-            dl_filename = os.path.join(self.pipeline_dir, "modules", "nf-core", filename)
+            split_filename = filename.split("/")
+            dl_filename = os.path.join(self.pipeline_dir, "modules", *install_folder, *split_filename[1:])
```

Comment:
This was done because `filename` has the format `software/vcftools/functions.nf`, so the extra `software` has to be removed.